svthalia/concrexit
# Replace django compressor

Closes #3200.



### Summary
This replaces compressor with https://github.com/jrief/django-sass-processor. 
Note that django-sass-processor actually does more or less depend on compressor (and libsass) but only at build-time.

It looks like this also opens up S3 for staticfiles very nicely.

### How to test
I'll deploy to staging to check if it works nicely. I've modified the workflow to be able to do that manually from a PR instead of master.
